docs/docs/workflows/config.md

```diff
@@ -41,7 +41,7 @@ constellation config generate aws
 This creates the file `constellation-conf.yaml` in the current directory.
 
 :::tip
-You can also automatically generate a configuration file by adding the `--generate-config` flag to the `constellation iam create` command when [creating an IAM configuration](#creating-an-iam-configuration).
+You can automatically update a configuration file with your IAM configuration by adding the `--update-config` flag to the `constellation iam create` command when [creating an IAM configuration](#creating-an-iam-configuration).
 :::
 
```

Comment:
The tip can be completely removed as the `config generate` step is now mandatory. It's sufficient that the flag is explained in the other section.